client/src/ui/styles/ManuscriptUpload.scss

```diff
@@ -0,0 +1,10 @@
+
+.manuscript-upload {
+    border: 0.063rem solid #e0e0e0;
+    width: 100%;
+
+    &__dropzone {
```

Comment:
We've been following the convention of complete class paths with styling rather than nesting like this.

Reply:
why? its not actually nesting in the output css:
```
.manuscript-upload {
  border: 0.063rem solid #e0e0e0;
  width: 100%; }
  .manuscript-upload__dropzone {
    padding: 2.25rem;
    text-align: center; }
```